_data/banking.yml

```
@@ -649,9 +649,10 @@ websites:
     - name: MBNA Canada
       url: https://www.mbna.ca
       img: mbnacanada.png
-      tfa: No
-      twitter: MBNA_Canada
-      facebook: MBNACanada
+      tfa: Yes
+      sms: Yes
+      phone: Yes
+      doc: https://www.mbna.ca/help-centre/credit-cards/two-step-verification-faq/index.jsp
```

```suggestion
      doc: https://www.mbna.ca/help-centre/credit-cards/two-step-verification-faq/
```
`index.jsp` is superflous in that link.